docs/decisions/20230509-message-routing-for-multi-tenant.md

```diff
@@ -1,24 +1,24 @@
 # Routing Requests to the Correct Tenant
 
-- Status: draft [ accepted | deprecated | superseded by [xxx](yyyymmdd-xxx.md)]
+- Status: accepted
```

Comment:
Please explain the reason for the change from draft to accepted within the renaming job.

Reply:
This ADR was also accepted and implemented, so it should be in `accepted` status.